apps/air-discount-scheme/backend/migrations/20201001142253-fix-financial-state-default-value.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface) => {
+    return queryInterface.sequelize.query(`
+      BEGIN;
+
+        ALTER TABLE flight_leg ALTER COLUMN financial_state SET DEFAULT 'AWAITING_DEBIT';
```

Comment:
do you also need to `update table flight_leg set financial_state='AWAITING_DEBIT' where financial_state='awaitingDebig';`?

Reply:
Nope.. no records for it -> thankfully handled in the app layer.

However, it doesn't hurt to be extra careful and add it.

Reply:
No we checked with the lambda thing, there were no records with `awaitingDebit`

Reply:
:hindu_temple: :church: